feelslikemonday/app/src/main/java/com/example/feelslikemonday/DAO/UserDAO.java

```diff
@@ -68,4 +78,25 @@ public void onFailure(@NonNull Exception e) {
                     }
```

Comment:
Same thing as above with converting e to a string- like I said this is just something I noted I have no idea if it actually matters or not